### PR TITLE
Replace bufferSize with highWatermark.

### PR DESCRIPTION
**Pull request by Nibbler999**
bufferSize was removed from node in 0.9.12 so this test was ineffective

### CODE REVIEW

File: test/WebSocket.test.js [hunk 1]
```diff
@@ -928,8 +928,7 @@ describe('WebSocket', function() {
         var ws = new WebSocket('ws://localhost:' + port);
         var callbackFired = false;
         ws.on('open', function() {
-          var fileStream = fs.createReadStream('test/fixtures/textfile');
-          fileStream.bufferSize = 100;
+          var fileStream = fs.createReadStream('test/fixtures/textfile', { highWaterMark: 100 });
```
Comment (lpinca) [inline test/WebSocket.test.js]: Should be `100 * 1024` for 100kb.

File: test/WebSocket.test.js [hunk 2]
```diff
@@ -953,9 +952,8 @@ describe('WebSocket', function() {
         var ws = new WebSocket('ws://localhost:' + port);
         var callbackFired = false;
         ws.on('open', function() {
-          var fileStream = fs.createReadStream('test/fixtures/textfile');
+          var fileStream = fs.createReadStream('test/fixtures/textfile', { highWaterMark: 100 });
```
Comment (lpinca) [inline test/WebSocket.test.js]: You might also want to pass the `encoding` as an option?

Comment (Nibbler999) [inline test/WebSocket.test.js]: Added

File: test/WebSocket.test.js (hunk 1 above)
Comment (Nibbler999) [inline test/WebSocket.test.js]: Why? I'm pretty sure 100 bytes is correct here.

Comment (lpinca) [inline test/WebSocket.test.js]: Yeah my bad. It's 100 bytes not 100kb, ignore my comment :)

**Comment by JoshuaWise** [comment]
Thanks!

**Pull request merged**